<launch>
  <node name="pick_objects" pkg="pick_objects" type="pick_objects" output="screen">
    <!-- Initial position -->
    <param name="/init_marker_pose_x" type="double" value="1.3" />
    <param name="/init_marker_pose_y" type="double" value="-5.53" />
    <param name="/init_marker_pose_z" type="double" value="0.0" />
    
    <param name="/init_marker_orient_x" type="double" value="0.0" />
    <param name="/init_marker_orient_y" type="double" value="0.0" />
    <param name="/init_marker_orient_z" type="double" value="0.0" />
    <param name="/init_marker_orient_w" type="double" value="1" />


    <!-- Final position -->
    <param name="/dest_marker_pose_x" type="double" value="-4.44" />
    <param name="/dest_marker_pose_y" type="double" value="1.33" />
    <param name="/dest_marker_pose_z" type="double" value="0.0" />
    
    <param name="/dest_marker_orient_x" type="double" value="0.0" />
    <param name="/dest_marker_orient_y" type="double" value="0.0" />
    <param name="/dest_marker_orient_z" type="double" value="0.0" />
    <param name="/dest_marker_orient_w" type="double" value="1" />
  </node>
</launch>
